app/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+
+from api.apod import apod_api
+from auth import auth
+
+from db import db
+app = Flask(__name__)
+
+app.config['SECRET_KEY'] = 'asd[jdsvcx3'
```

Comment:
It seems to me that it is not very safe to store the secret key in app.py file. Perhaps you should move it to environment variables?

I also have a suggestion to put all app.config values into a separate Config file

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/e7a312b2874fcbc81eb477d9ba5760e56419ee07